### PR TITLE
Updating CHANGELOG.md for release 0.3.1

### PR DESCRIPTION
**Pull request by shachafl**
The master branch is protected against updates.
Changes must be made through a pull request.

### CODE REVIEW

**Comment by shachafl** [comment]
@berl I needed to do a PR to merge the update to the CHANGELOG.md prior to the new release.
Can you please approve?

File: CHANGELOG.md [hunk 1]
```diff
@@ -273,6 +280,7 @@
 ## [0.0.33] - 2019.02.14
 - Last release without a changelog!
 
+[0.3.1]: https://github.com/spacetx/starfish/releases/tag/0.3.0
```
Comment (berl) [inline CHANGELOG.md:283]: should this be 0.3.1 in the link?

**Review by berl — APPROVED** [review]
looks good, fix the version tag in the link if needed

File: CHANGELOG.md (hunk 1 above)
Comment (shachafl) [reply CHANGELOG.md:283]: You are correct.
Thanks for catching that.

**Pull request merged**